esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -896,14 +1010,42 @@ EsdlBindingImpl::EsdlBindingImpl(IPropertyTree* cfg,
     }
 }
 
-bool EsdlBindingImpl::reloadDefinitions(IPropertyTree * esdlBndCng)
+void EsdlBindingImpl::saveDESDLState()
 {
-    if ( m_pESDLService)
+    try
+    {
+        //rodrigo is this the best way to fetch the ESDL def for a given service???
+        Owned<IEsdlDefObjectIterator> it = m_esdl->getDependencies(m_espServiceName.str(), "", 0, NULL, 0);
```

Comment:
Hmmm.  Should we reduce our m_esdl to only these objects in the first place?  If we are doing the dependencies gathering right we shouldn't need anything else, right?